Add OneDark theme

azrikahar opened the pull request:
I was looking for One Dark theme (became my go-to after having used Atom) but seems like there is only a derived theme called One Half Dark and I wished to have the original one. I thought perhaps others may want to have it as well.

With that said, seems like you are getting the colors from iTerm2, so maybe this One Dark theme needs to be added to your script as well? Perhaps a simple Array.push,  or maybe concat the final array from another custom json file.

atomcorp commented:
Hey, thanks so much for contributing!

The theme json is generated from a script, `script/get-schemes.js`, so the next time I run it your changes would be overwritten. I think as you suggested there should be a custom themes json file in the src folder that is merged with the one build from the iTerm2 repo.

You're more than welcome to do that! If not, no worries, I'd beeen thinking about improving a couple of other things so I can add it to the list.

Cheers again!

azrikahar commented:
Hi, just added the custom themes json file as we discussed. I have also added sorting, though on hindsight I'm not sure if it's necessary since you already didn't sort them in the final output on the interface.

atomcorp commented:
That's great! Thanks so much for the idea and implementation!
